types/asana/index.d.ts

```diff
@@ -4,6 +4,7 @@
 //                 Tasyp <https://github.com/tasyp>
```

Comment:
Probably should update from 0.18.4 to 0.18.14, since these changes seem comprehensive. (It won't make a difference to the `@types/asana` version but it's at least nice to have the header be correct.)